release/xgboost_tests/xgboost_tests.yaml

```diff
@@ -8,9 +8,10 @@
     compute_template: tpl_cpu_small.yaml
 
   run:
+    use_connect: True
     timeout: 600
     prepare: python wait_cluster.py 4 600
-    script: python workloads/train_small.py
+    script: workloads/train_small.py
```

Comment:
Doesn't this break execution as we don't have a shebang?

Reply:
I was thinking of handling `script` different when run as client, by passing `script` as one arg among the list of `args` to `subprocess.Run()`. Reverted now.